2022-10-10-report.org

```diff
@@ -0,0 +1,96 @@
+#+title:  Weekly report for 3rd to 10th October 2022
+#+author: Philip Herron and Arthur Cohen
+#+date:   2022-10-10
+
+** Overview
+
+Thanks again to [[https://opensrcsec.com/][Open Source Security, inc]] and [[https://www.embecosm.com/][Embecosm]] for their ongoing support for this project.
+
+*** Milestone Progress
+
+This week we returned to a more usual level of productivity, and achieved a lot of work on the intrinsics side of things as well as our exporting mechanism. We spent some time fixing the early reviews we got from our upstream patches and are working on the next set of patches.
+
+We have put together a planning for the foreseeable future of the project and would love to hear your thoughts about it: https://hackmd.io/@RebJtN4VQzOcP4MsOf8inA/SkjK2UOfs
+Feel free to raise issues in the `gccrs` repository to discuss anything!
+
+Finally, the project is also accepting hacktoberfest contributions and we would love to mentor anyone interested in contributing! Feel free to start working on one of the issues marked `good-first-pr` or to join our [[https://gcc-rust.zulipchat.com/][Zulip]] and ask for work.
+
+** Completed Activities
+
+- AST: dump structs, unions and enums [[https://github.com/rust-gcc/gccrs/pull/1572][PR1572]]
+- Implement data prefetch intrinsics [[https://github.com/rust-gcc/gccrs/pull/1565][PR1565]]
+- builtins: Move implementation into source file [[https://github.com/rust-gcc/gccrs/pull/1564][PR1564]]
+- ast: dump assignment and if expressions [[https://github.com/rust-gcc/gccrs/pull/1563][PR1563]]
+- Support looking up super trait items during path resolution [[https://github.com/rust-gcc/gccrs/pull/1562][PR1562]]
+- Fix -Wunused using `rust-lang.cc` hack [[https://github.com/rust-gcc/gccrs/pull/1560][PR1560]]
+- Fix include structure [[https://github.com/rust-gcc/gccrs/pull/1559][PR1559]]
+- testing: try loop in const function [[https://github.com/rust-gcc/gccrs/pull/1554][PR1554]]
+- Dump item visibility [[https://github.com/rust-gcc/gccrs/pull/1547][PR1547]]
+- Reintroduce targetrustm hooks without the maze of macro definitions in target headers [[https://github.com/rust-gcc/gccrs/pull/1543][PR1543]]
+
+*** Contributors this week
+
+- [[https://github.com/ibuclaw][Iain Buclaw]]
+- [[https://github.com/dafaust][David Faust]]
+- [[https://github.com/abbasfaisal][Faisal Abas]]
+
```

Comment:
I think we missed @dkm lets add him in :D

Reply:
Sure, forgetting such a big contribution would be sad :D Thanks anyway!